src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const editWords = (words) => words.trim();
+  const editArray = sourceString
+    .split(';')
+    .filter((el) => el !== '' && el !== '  ;')
```

Comment:
[CODE PRACTICE] - Instead of filtering out empty strings and strings with '  ;', consider using a more robust check for non-empty CSS declarations. You could use a regular expression to match valid CSS declarations.